frontend/src/components/submission/SubmissionForm.vue

```diff
@@ -39,9 +40,19 @@ const assigneeOptions: Ref<Array<User>> = ref([props.submission.user]);
 // Default form values
 const initialFormValues: any = {
   ...props.submission,
+  applicationStatus: props.submission.applicationStatus
+    ? props.submission.applicationStatus
+    : APPLICATION_STATUS_LIST.NEW,
```

Comment:
We should be setting 'New' when the entry is created in the database. This just masks the issue until they manually resave the data.